cli_tools/gce_ovf_import/ovf_utils/ovf_programmatic_units.go

```diff
@@ -0,0 +1,49 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Why "programmatic_units" for the file name?

As is, the name for this file should be "ovf_unit_parser" or similar.

Reply:
It's the vernacular used in the OVF 2.x spec:

```
Values for ovf:capacityAllocationUnits shall match the format for
programmatic units defined in DSP0004 with the restriction that
the base unit shall be "byte".
```


https://www.dmtf.org/sites/default/files/standards/documents/DSP0243_2.1.1.pdf